packages/@aws-cdk-testing/framework-integ/test/pipelines/test/integ.newpipeline-with-vpc.ts

```diff
@@ -1,14 +1,16 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 /// !cdk-integ PipelineStack pragma:set-context:@aws-cdk/core:newStyleStackSynthesis=true
-import * as path from 'path';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { App, Stack, StackProps, Stage, StageProps } from 'aws-cdk-lib';
 import * as ec2 from 'aws-cdk-lib/aws-ec2';
 import * as s3_assets from 'aws-cdk-lib/aws-s3-assets';
 import * as sqs from 'aws-cdk-lib/aws-sqs';
-import { App, Stack, StackProps, Stage, StageProps } from 'aws-cdk-lib';
-import { Construct } from 'constructs';
 import * as pipelines from 'aws-cdk-lib/pipelines';
+import { Construct } from 'constructs';
+import * as path from 'path';
 import { PIPELINE_REDUCE_ASSET_ROLE_TRUST_SCOPE } from 'aws-cdk-lib/cx-api';
 
+
```

Comment:
Build is yelling at you here for having multiple blank lines.

Reply:
has been resolved